hazelcast/src/main/java/com/hazelcast/map/impl/operation/PartitionCheckIfLoadedOperation.java

```diff
@@ -42,14 +44,26 @@ public PartitionCheckIfLoadedOperation(String name, boolean doLoad) {
         this.doLoad = doLoad;
     }
 
+    public PartitionCheckIfLoadedOperation(String name, boolean doLoad, boolean waitForKeyLoad) {
```

Comment:
just a small one: the other 2 constructors should call this one. 
```java
    public PartitionCheckIfLoadedOperation(String name) {
        this(name, false);
    }

    public PartitionCheckIfLoadedOperation(String name, boolean doLoad) {
        this(name, doLoad, false);
    }

    public PartitionCheckIfLoadedOperation(String name, boolean doLoad, boolean wait) {
        super(name);
        this.doLoad = doLoad;
        this.waitForKeyLoad = waitForKeyLoad;
    }
```

I think it makes code cleaner.